packages/test-data/less/_main/media.less

```diff
@@ -238,4 +238,68 @@
   .selector {
     foo: bar;
   }
-}
\ No newline at end of file
+}
+
+@media screen and (color), projection and (color) {
+  .selector {
+    color: #eee;
+  }
+}
+
+@media not (width <= -100px) {
+  body { 
+    background: green; 
+  }
+}
+
+@media (height > -100px) {
+  body { 
+    background: green; 
+  }
+}
+
+@media not (resolution: -300dpi) {
+  body { 
+    background: green; 
+  }
+}
+
+@media (example, all,), speech { 
```

Comment:
Was this already allowed for Less media queries? This would be against the CSS spec AFAIK.

Reply:
Fix my pull request to be consistent with Less.js version 4.1.3. Thank you for reviewing. See latest comment.